src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,10 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
```

Comment:
Ensure that the `comments` prop is correctly passed to the `CommentList` component. This prop should be an array of comment objects.